scripts/mutect2_wdl/unsupported/calculate_sensitivity.py

```diff
@@ -82,9 +82,16 @@ def assign_indel_af_bin(af):
                               columns = flatten([(str(x) + "_LCI", str(x) + "_UCI") for x in af_bins]))
 
     sensitivity_with_confidence = sensitivity_table.merge(error_bars, left_index = True, right_index = True)
-    sensitivity_with_confidence.to_csv( snp_or_indel + "_sensitivity.tsv", sep = '\t')
 
-    return sensitivity_with_confidence, af_bins
+    # put in the raw number of calls
```

Comment:
Can I have a little bit of a higher level description of what you are doing?  Both in this code here and in general?  I don't know there specifics of what was requested...